db/migrate/20170116210702_add_manager_ref_to_authentications.rb

```diff
@@ -0,0 +1,5 @@
+class AddManagerRefToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :manager_ref, :string
```

Comment:
Agreed.  I'll merge it and broadcast what I think the follow up changes should be.